### PR TITLE
updated the readme

### PR DESCRIPTION
**Pull request by mvpache**
Updated the readme to more clearly explain stuff

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,4 @@
-# pr-example
\ No newline at end of file
+# pr-example is so cool
+
+more read me notes  
+what is the meaning of life
```
Comment (mvpache) [inline README.md:4]: Was feeling a little existential

Comment (mvpache) [reply README.md:4]: OKay but now is not the time for jokes

**Comment by mvpache** [comment]
Was fixed by PR#123

**Comment by mvpache** [comment]
@JuliusDorfman when are you reviewing this PR, michael is gonna fail us

**Comment by JuliusDorfman** [comment]
I'll get to it when I get to it

**Review by JuliusDorfman — COMMENTED** [review]
Looks good to me!

**Review by JuliusDorfman — APPROVED** [review]

**Pull request merged**